content/blog/2024/04/2024-04-04-gsoc-2024-contributor-application-period-concludes.adoc

```diff
@@ -0,0 +1,26 @@
+---
+layout: post
+title: "Post Google Summer of Code 2024 contributor application period"
+tags:
+- gsoc
+- gsoc2024
+- events
+- community
+authors:
+- krisstern
+sig: gsoc
+description: >
+  Some info on the selection process of the Google Summer of Code 2024 contributor applications for Jenkins.
+opengraph:
+  image: /images/gsoc/opengraph.png
+---
+
+image:/images/gsoc/jenkins-gsoc-logo_small.png[Jenkins GSoC, role=center, float=right]
+
+The Google Summer of Code 2024 contributor application period has ended, and we are excited to have received a number of more than 75 valid applications from potential contributors interested in contributing to Jenkins via this program. We are grateful to all the students who applied and are looking forward to reviewing their applications over the next few weeks.
+
+Over the past month or so between March 18 2024 when the contributor application period opened and the end of the application period on April 2nd, 2024, the Jenkins project has been busy fielding candidate questions about the program, as well as reviewing some 40 odd draft proposals for GSoC hopefuls who have expressed interest in applying for the program. While due to privacy reasons we cannot share every detail about the ranking process that is done at Jenkins, we can share some insights into the process in general. Normally all potential mentors are invited to grade each of the submitted proposals on a scale of 1 to 5, with 1 being a strong rejection and 5 being a strong acceptance. The Jenkins GSoC org admins then take the average of all the scores for each proposal and use that to rank the proposals. The top ranked proposals are then nominated by Jenkins to Google for further consideration. There might be some slight variations in this process, in that we also consider how each applicant interacts with fellow applicants during (and maybe even beyond) the application period, as well as how they interact with the Jenkins community in general. All-in-all, the ideal applicant would be something with an excellent proposal, usually one with a global grade of between 4 and 5, who has shown a willingness to engage with the Jenkins community, and who has shown a willingness to learn and grow as a contributor. Essentially, we would like to select the proposals that will have a high likelihood of succeeding.
```

Comment:
```suggestion
Over the past month or so, between March 18 2024 when the contributor application period opened and the end of the application period on April 2nd, 2024, the Jenkins project has been busy fielding candidate questions about the program, as well as reviewing some 40 odd draft proposals for GSoC hopefuls who have expressed interest in applying for the program.
While due to privacy reasons we cannot share every detail about the ranking process that is done at Jenkins, we can share some insights into the process in general.
Normally, all potential mentors are invited to grade each of the submitted proposals on a scale of 1 to 5, with 1 being a strong rejection and 5 being a strong acceptance.
The Jenkins GSoC org admins then take the average of all the scores for each proposal and use that to rank the proposals.
The top ranked proposals are then nominated by Jenkins to Google for further consideration.
There might be some slight variations in this process, in that we also consider how each applicant interacts with fellow applicants during (and maybe even beyond) the application period, as well as how they interact with the Jenkins community in general.
All-in-all, the ideal applicant would be something with an excellent proposal, usually one with a global grade of between 4 and 5, who has shown a willingness to engage with the Jenkins community, and who has shown a willingness to learn and grow as a contributor.
Essentially, we would like to select the proposals that will have a high likelihood of succeeding.
```

Reply:
The choice of the word "privacy" in "While due to privacy reasons we cannot share every detail about" is bothering me. Would "While we can't share all the details of the ranking discussions to preserve the serenity of the debates, we can..."

Reply:
I would also mention that a strong mentoring team with an experienced lead-mentor plays also an important parameter in the project/proposal ranking